All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**Edson Ayllon**
+We could just start opening up a PR to EIP 1. There's nothing stopping us from making future PRs. Yep.
+
+**James Hancock**
+Yeah, I've been hesitant to dump into the network process. I'd just like to get this standardization process at least more moving forward before I start diverting efforts. But I'm open to feedback from the group on what's the best way to proceed there.
+
+**Pooja Ranjan**
+One of the action items mentioned, or not an action item - it's just a part of the decision is like that there should be a particular place to collect all these kind of definitions that we are planning. There is already a mandatory terminology section for EIP. So we are discussing for putting these statuses as described in EIP. It is something that we should consider for EIPs in general as well. If something new is coming up, we should add this section in an EIP. I'm talking about an EIP template that we create.
+
+**Edson Ayllon**
+Yeah, I think we mentioned this in the previous meeting. I had suggested that different standard organizations, they have a section for terminology. It might be good if we simplify to something like, say, definitions as having it mandatory. There may be EIPs that don't really introduce new terms. And then having global terms that are required for each EIP, I don't think is necessary. But some  EIPs may introduce new terminology that would facilitate clarifying to the reader. It brings some value having a definition section.
+
+**James Hancock**
+Yeah, I would agree.
+
+**Pooja Ranjan**
+I'm not too sure how much value it would add if it is not something that is going to be commented. And in most of the EIPs, we can probably go ahead and explain that term in one of the summary sections or somewhere else in the EIP itself. But adding this as a mandatory section for every EIP that is something I'm not sure about, but it doesn't hurt to have something additional as well.
+
+**Edson Ayllon**
+So I think it's good as an optional section. I don't think that it needs to be mandatory for every single EIP.
+
+**James Hancock**
+Yeah, it would be good to have one in some of them going to EIP 1. We can have definitions that people can use, like the definition of a network upgrade.
+
+**Pooja Ranjan**
+Right, that is that is needed for EIP one. I understand this point. This section is mandatory for EIP 1. But again, EIP 1 is not any general EIP. It's a kind of documentation that provides a guideline. So for that, having this section, it makes a lot of sense. But making it mandatory, I was not sure. But if it is optional, yeah, perfectly fine.
```

Comment:
Right, that is needed for EIP-1. I understand this point. This section is mandatory for EIP-1. But again, EIP-1 is not any general EIP. It's a kind of documentation that provides a guideline. So for that, having this section, it makes a lot of sense. But making it mandatory, I was not sure. But if it is optional, yeah, perfectly fine.